server/routes/createGoal/createGoalController.ts

```diff
@@ -65,6 +66,7 @@ export default class CreateGoalController {
     const { prisonerSummary } = req.session
     const { createGoalForm } = req.session
     const addStepForm = req.session.addStepForm || { stepNumber: 1 }
+    req.session.addStepForms = req.session.addStepForms || new Array<AddStepForm>()
```

Comment:
```
req.session.addStepForms = req.session.addStepForms || []
```
Would that work?

Reply:
Yep. I understand more about how type safety works (or doesn't work) in TS.